.config/configuration.winget

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://aka.ms/configuration-dsc-schema/0.2
+# Reference: https://github.com/microsoft/terminal/blob/main/README.md#developer-guidance
+properties:
+  resources:
+    - resource: Microsoft.Windows.Developer/DeveloperMode
+      directives:
+        description: Enable Developer Mode
+        allowPrerelease: true

```

Comment:
```suggestion
        allowPrerelease: true
        securityContext: elevated
```

When a resource needs to run "elevated" the security context will provide the hint to WinGet to prompt for UAC during agreement acceptance. Only resources decorated this way run elevated. In the case of "Developer Mode", an unelevated "test" in DSC will succeed, but if the setting actually needs to be enabled, it will fail without elevation.

Reply:
https://github.com/microsoft/winget-cli/blob/a3cea271bb84cf2a3358d97242effc348cce18a5/schemas/JSON/configuration/configuration.schema.0.2.json#L77-L81